apify-docs-theme/src/theme.js

```diff
@@ -132,6 +132,21 @@ This either means that your Docusaurus setup is misconfigured, or that your GitH
             o.plugins.push(postcssPreset); // allow newest CSS syntax
             return o;
         },
+        injectHtmlTags() {
```

Comment:
Questing: @webrdaniel can't we use the https://www.npmjs.com/package/@gracefullight/docusaurus-plugin-intercom plugin for this?

Reply:
I was kinda discouraged by only 12 weekly downloads but yeah we can use this 👍

Reply:
I have some issues getting it working with the plugin. It is producing an invalid script. So I guess we will stick with the manual implementation
<img width="559" alt="Screenshot 2024-09-02 at 11 53 41" src="https://github.com/user-attachments/assets/48feb957-2ebe-4b04-8fd0-2bb50204a123">